src/tiles/image-block.ts

```diff
@@ -106,6 +106,12 @@ export class ImageBlock extends LitElement {
       }
 
       /** tile-list view */
+      .list {
+        border-radius: 0;
+        background-color: var(--imageBlockListBackgroundColor, #ebebee);
```

Comment:
`#ebebee` is new to me! works for me if @pezvi approved 👍

Reply:
Oh yeah, I meant to check if there's a better color already in use that would fit here. I'll talk to @pezvi & adjust accordingly. Thanks! 🐝